pkg/serverutils/knative_event_source_utils.go

```diff
@@ -0,0 +1,75 @@
+package serverutils
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+// CustomResourceDefinitionSpec describes how a user wants their resource to appear
+type CustomResourceDefinitionSpec struct {
```

Comment:
no need to redefine.. the type already exists https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/types.go#L34